mitodl/ocw-studio
# Upgrade mitol-django-authentication, common, pygithub

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #23 

#### What's this PR do?
Upgrades some packages as a prerequisite to installing mitol-django-mail. These changes should be simple upgrades without a change of functionality so it should hopefully be easy to spot and fix if this breaks something.

#### How should this be manually tested?
- Rebuild the docker container so that the latest packages are installed
- Follow the instructions here to test SSO login: https://github.com/mitodl/ocw-studio#testing-touchstone-login-with-saml-via-ssocircle. I can help walk you through it if setting this up is unfamiliar to you
- Follow the manual testing instructions in this PR to test that github syncing still works: https://github.com/mitodl/ocw-studio/pull/216. You should be able to create a website, and sync and delete content.
